src/ExpressionEvaluator/Core/Test/ExpressionCompiler/ExpressionCompilerTestHelpers.cs

```diff
@@ -840,7 +840,7 @@ internal static unsafe MetadataReader ToMetadataReader(this PEMemoryBlock metada
                         comp.MessageProvider,
                         () => peStream,
                         () => pdbStream,
-                        null, null,
+                        null, null, null,
```

Comment:
Can we make this third `null` a named argument?

Reply:
Done